tools/rustfmt/BUILD.bazel

```diff
@@ -60,7 +60,7 @@ rust_binary(
 )
 
 rust_clippy(
-    name = "rustfmt_clippy",
+    name = "clippy",
```

Comment:
Why this change?

Reply:
I don't think there should be a need for `package:package_thing` targets. They should just be `package:thing`. But I can remove this if you'd prefer.

Reply:
Makes sense to me, but let's make a separate PR for that?

Reply:
Reverted though.